core/src/main/java/apoc/text/Strings.java

```diff
@@ -96,7 +95,12 @@ public long byteCount(final @Name("text") String text, @Name(value = "charset",
     @UserFunction
     @Description("apoc.text.bytes(text,[charset]) - return bytes of the text")
     public List<Long> bytes(final @Name("text") String text, @Name(value = "charset", defaultValue = "UTF-8") String charset) throws UnsupportedEncodingException {
-        return convertFromBytesToList(text.getBytes(charset));
+        byte[] bytes = text.getBytes(charset);
+        List<Long> result = new ArrayList<>(bytes.length);
+        for (byte b : bytes) {
+            result.add((long)b & 0xFFL);
```

Comment:
forgive my ignorance here, why is the bitwise operation needed here? 😅

Reply:
Well, actually this code was not written by me 😅.
I created the util function `convertFromBytesToList()` using this piece of code to convert from `byte[]` to `List<Long>` in pr #1746.
Now, I don't need this anymore because, as in description, the two functions `apoc.util.compress/decompress` should use `byte[]` and not `List <Long>`, so I put the function back as before the merge.

Reply:
Well, actually this code was not written by me 😅
I created a util function using this piece of code to convert from `byte[]` to `List<Long>` in pr #1746.
Now, I don't need this anymore because, as in description, the two functions `apoc.util.compress/decompress` should use `byte[]` and not `List<Long>`, so I put the function back as before the merge.